clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -437,7 +453,409 @@ void SemaHLSL::handleShaderAttr(Decl *D, const ParsedAttr &AL) {
     D->addAttr(NewAttr);
 }
 
+struct RegisterBindingFlags {
+  bool Resource = false;
+  bool Udt = false;
+  bool Other = false;
+  bool Basic = false;
+
+  bool Srv = false;
+  bool Uav = false;
+  bool Cbv = false;
```

Comment:
UDT, SRV, UAV, and CBV are all initialisms, it probably makes the most sense to fully capitalize them.

Reply:
👍